.circleci/config.yml

```diff
@@ -34,14 +34,47 @@ jobs:
             git tag $TAG_NAME
             git push origin $TAG_NAME
 
+  integration-test: # should this be called something else? or maybe rename build? since build is basically just testing?
+    docker:
+      - image: circleci/node:8.11
+
+    environment:
+      IMAGE_NAME: quay.io/broadinstitute/martha
+      QUAY_USER: broadinstitute+martha_circle_bot
+
+    steps:
+      - checkout
+
+      - setup_remote_docker:
+          docker_layer_caching: true
+
+      - run:
+          name: Build image
+          command: |
+            docker build -t ${IMAGE_NAME}:${CIRCLE_BRANCH} -f docker/Dockerfile .
+            if [[ ${CIRCLE_BRANCH} == "dev" ]]; then docker tag ${IMAGE_NAME}:${CIRCLE_BRANCH} ${IMAGE_NAME}:latest ; fi
+
+#      - run:
```

Comment:
is the commented out code still valid?

Reply:
I ended up making the changes in a different branch which can be seen [here, in another PR](https://github.com/broadinstitute/martha/pull/113). I'll port those changes over into this PR before I merge this PR.